lib/pbench/test/unit/server/query_apis/test_controllers_list.py

```diff
@@ -183,49 +168,39 @@ def test_empty_query(
 
         index = self.build_index(server_config, ("2020-08", "2020-09", "2020-10"))
         response = query_api(
-            json, index, HTTPStatus.OK, server_config, json=response_payload
+            "/controllers/list",
+            "/_search?ignore_unavailable=true",
+            json,
+            index,
+            HTTPStatus.OK,
+            status=HTTPStatus.OK,
+            json=response_payload,
         )
-        res_json = response.json
-        assert res_json == []
+        assert response.json == []
 
     @pytest.mark.parametrize(
         "exceptions",
         (
             {
-                "exception": make_http_exception(HTTPStatus.BAD_REQUEST),
+                "exception": requests.exceptions.ConnectionError(),
                 "status": HTTPStatus.BAD_GATEWAY,
             },
             {
-                "exception": requests.exceptions.ConnectionError,
-                "status": HTTPStatus.BAD_GATEWAY,
-            },
-            {
-                "exception": requests.exceptions.Timeout,
+                "exception": requests.exceptions.Timeout(),
                 "status": HTTPStatus.GATEWAY_TIMEOUT,
             },
             {
-                "exception": requests.exceptions.InvalidURL,
+                "exception": requests.exceptions.InvalidURL(),
                 "status": HTTPStatus.INTERNAL_SERVER_ERROR,
             },
-            {"exception": Exception, "status": HTTPStatus.INTERNAL_SERVER_ERROR},
+            {"exception": Exception(), "status": HTTPStatus.INTERNAL_SERVER_ERROR},
```

Comment:
Maybe, but that wasn't a change in this PR, and any choice of a specific `Exception` instance aside from the ones we expect would be imaginary anyway. We really do intend this as a generic catch-all.

Reply:
> We really do intend this as a generic catch-all.

Actually, no:  we expect the **_CUT_** to operate as a generic catch-all, which we would prove by raising something _else_ here.

And, it almost doesn't matter what we raise (so long as it should match) -- all we want to do is prove that the catch-all code responds reasonably to an otherwise unexpected exception.  So, while raising `Exception` certainly suffices, it's inferior, because that's what the code is _expecting_....  ;-)